mips/pmap.c

```diff
@@ -347,7 +348,7 @@ void tlb_exception_handler(exc_frame_t *frame) {
       log("Address %08lx not mapped by any active pmap!", orig_vaddr);
       goto fault;
     }
-    if (is_valid(pmap->pde[index])) {
+    if (pmap->pde && is_valid(pmap->pde[index])) {
```

Comment:
BTW. Yesterday, during a meeting, we identified a bug for which a quick workaround would be to flush TLB during `pmap_activate`. We've worked out a better solution, but @strzkrzysiek changes are not ready to integrate with master.